java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -72,6 +72,7 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
```

Comment:
Yeah I think I added that because IntelliJ was giving me a warning 😅 I will switch the place I used `@Nonnull` to use `@Nullable` even if that means a little bit of duplication between the two constructors, it's more technically correct anyway.

Reply:
There is no need to switch `@Nonnull` to `@Nullable`. My point here is that, for us, no annotation should mean non-null by default (I still need to enroll Piranha in NullAway to make that an actual compiler check, but the coding guideline remains)